BUFR_TableD_en_40.csv

```diff
@@ -793,3 +793,169 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,022130,Number of valid points for specific band,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,025167,Specific band net instrumental correction,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,025163,Altimeter ionospheric correction on ku band,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,015012,Total electron count per square metre,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,025164,Radiometer wet tropospheric correction,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,013090,Radiometer water vapour content,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,013091,Radiometer liquid content,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,025166,Sea state bias correction on specific band,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,007002,Height or altitude,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,011097,Wind speed from altimeter,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021183,Specific band corrected ocean backscatter coefficient,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021184,Std specific band corrected ocean backscatter coefficient,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,022134,Number of valid points for specific band backscatter,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021122,Attenuation correction on sigma-0 (from tb),,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021186,Specific band automatic gain control,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021187,Rms specific band automatic gain control,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021188,Number of valid points for specific band automatic gain control,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,201131,Change data width,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,021185,Specific band net instrumental correction for agc,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,201000,Reset width,,,Operational
+40,Additional satellite report sequences,340023,(1 Hz C and Ku band values),,022192,Specific band significant wave height,,,Operational
```

Comment:
This descriptor has been renumbered by Secretariat to be 0-22-179, please adjust accordingly.